### PR TITLE
Implement Hamming distance exercise

### PR DESCRIPTION
**Pull request by marnen**
This is the [Hamming distance](https://github.com/exercism/problem-specifications/tree/master/exercises/hamming) exercise. I love how concise it is in APL!

@kytrinyx if you want to review, feel free, but at this point, I think I'll probably just start merging exercises in immediately as I do them, unless you'd rather get another pair of eyes on them.

### CODE REVIEW

File: test.apl [hunk 1]
```diff
@@ -18,6 +18,11 @@
   test¯warn test¯read_unicode filename
 ∇
 
+⍝ Evaluate APL code (as string) and return output (in original type) or error message (as string).
+∇z←test∆try apl_string
+  z←(↑¯1↑⎕EC apl_string)[1;]
```
Comment (marnen) [inline test.apl]: I may want to be a little smarter about detecting actual errors. I still need to find out more about APL's error handling.

Comment (marnen) [reply test.apl]: Yeah, I think I need to add a few things.

**Comment by marnen** [comment]
Still need to change a few things. Will reopen when ready to merge.

**Comment by marnen** [comment]
I think this is ready to merge now.

**Comment by kytrinyx** [comment]
@marnen I'm so sorry I've left you high and dry around here lately.

I love how concise this is in APL, too! It's beautiful ✨

**Pull request merged**